tests/test_core.py

```diff
@@ -7921,6 +7921,72 @@ def test_asyncify_indirect_lists(self, args, should_pass):
       if should_pass:
         raise
 
+  @needs_dylink
+  @no_memory64('TODO: asyncify for wasm64')
+  def test_asyncify_side_module(self):
+    self.set_setting('ASYNCIFY')
+    self.set_setting('EXIT_RUNTIME', 1)
+    self.emcc_args += ['-sASYNCIFY_IMPORTS=["my_sleep"]']
```

Comment:
Interesting... If I understand, the issue is that

1. Normally `ASYNCIFY_IMPORTS` just contains JS functions, because those are the imports.
2. With dynamic linking, **a wasm function in one module can be an import in another**, and so `ASYNCIFY_IMPORTS` must contain wasm functions too, when they are linked across modules and when they end up on the stack when pausing.

Is that right? If so, we should document it as it is not obvious. Adding a note to `./site/source/docs/porting/asyncify.rst` would be good.

Reply:
This still needs to be addressed.

Reply:
That's correct.
I've added [some explanation](https://github.com/emscripten-core/emscripten/pull/15893/files#diff-841425e3398e112fdcaea671f046d9febda581c2ef8eb42e0cb28930df3aaaed)